src/react-compat.js

```diff
@@ -95,8 +97,13 @@ if (REACT013) {
   // to list this as a dependency in package.json and have 0.13 work properly.
   // As a result, right now this is basically an implicit dependency.
   try {
-    // eslint-disable-next-line import/no-extraneous-dependencies
-    TestUtils = require('react-addons-test-utils');
+    if (REACT155) {
+      // eslint-disable-next-line import/no-extraneous-dependencies
+      TestUtils = require('react-dom/test-utils');
+    } else {
+      // eslint-disable-next-line import/no-extraneous-dependencies
+      TestUtils = require('react-addons-test-utils');
+    }
   } catch (e) {
     // eslint-disable-next-line no-console
     console.error(
```

Comment:
Does this need to be in the `try` block? The error logged in `catch` only mentions `react-addons-test-utils` so it might be confusing if requiring `react-dom/test-utils` throws for whatever reason.